05-UI-ReactJS/topic-3/src/components/AddMovie.js

```diff
@@ -34,27 +71,27 @@ class AddMovie extends React.Component {
       <div>
         <br/>
         <h2>Add Movie</h2><hr/>
-        <form onSubmit={this.handleSubmit.bind(this)}>
+        <form onSubmit={this.handleSubmit}>
           <div>
             <label>Title </label><br/>
-            <input type="text" ref="title" />
+            <input type="text" name="title" value={this.state.title} onChange={this.handleChange}/>
```

Comment:
You are using `this.state.title` and in other places you are using `this.state.newMovie.title`. Maybe there is your issue